### PR TITLE
feat: home-page

### PR DESCRIPTION
**Pull request by BelideNikhil**
made the home page.
Note that the resets are in the index.css and will separated in the next push.

image

[Preview Homepage](https://project-97x.netlify.app/)

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    height: 100%;
+}
+.categ-card .card-image img{
+    object-fit: cover;
+}
+.categ-card .card-overlay-content{
+    background-color:rgb(0,0,0,0.1)
+}
+.categ-card .card-overlay-content span{
+    background-color:rgb(0,0,0,0.75);
```
Comment (manojasarsa) [inline index.css]: You can beautify file so that space can be consistent for all properties.

**Review by manojasarsa — APPROVED** [review]
Home Page Looks Awesome!

**Review by adidoshi — APPROVED** [review]
Good to go, UI looks clean!

**Review by himadri2110 — APPROVED** [review]
Good to go!

File: index.css (hunk 1 above)
Comment (BelideNikhil) [reply index.css]: I'll do that, thank you!

**Review by himadri2110 — APPROVED** [review]

File: index.css [hunk 2]
```diff
@@ -55,10 +55,10 @@
     height: 5rem;
     background-color: var(--clr-whitesmoke);
```
Comment (bharati-21) [inline index.css:56]: A suggestion as Vikram sir mentioned - not to mention the variable's value in the name.

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged! Super clean UI 🚀

**Pull request merged**